R/FilterStateEmpty.R

```diff
@@ -135,32 +134,6 @@ EmptyFilterState <- R6::R6Class( # nolint
       list(
         keep_na = self$get_keep_na()
       )
-    },
-
-    #' @description
-    #' Set state
-    #'
-    #' @param state (`teal_slice`)\cr
-    #'  only the `keep_na` field will be considered; a non-NULL `selected` field raises an error
-    #' \itemize{
-    #'   \item{`keep_na` (`logical`)}{ defines whether to keep or remove `NA` values }
-    #' }
-    #'
-    #' @return NULL invisibly
-    set_state = function(state) {
-      if (!is.null(state$selected)) {
-        stop(
-          sprintf(
-            "All values in variable '%s' are `NA`. Unable to apply filter values \n  %s",
-            private$varname,
-            paste(state$selected, collapse = ", ")
-          )
-        )
-      }
```

Comment:
This method gives a very specific error message, are we sure we want to get rid of it?

Reply:
Oh, that makes sense :+1: